docs/core/compatibility/7.0.md

```diff
@@ -123,6 +123,8 @@ If you're migrating an app to .NET 7, the breaking changes listed here might aff
 | [MSBuild serialization of custom types in .NET 7](sdk/7.0/custom-serialization.md) | ❌ | ❌ | 7.0.100 |
 | [dotnet test: switch `-a` to alias `--arch` instead of `--test-adapter-path`](https://github.com/dotnet/sdk/issues/21389) | ❌ | ❌ | Preview 1 |
 | [dotnet test: switch `-r` to alias `--runtime` instead of `--results-dir`](https://github.com/dotnet/sdk/issues/21952) | ❌ | ❌ | Preview 1 |
+| [Preview .NET 7 SDK installation workloads can interfere with RTM workloads](https://github.com/dotnet/core/blob/main/release-notes/7.0/known-issues.md#70-projects-using-certain-workloads-dont-load-build-and-or-run-if-net-7-preview-sdk-workloads-are-installed) |✔️ | ✔️ | Preview 3 |
```

Comment:
@PriyaPurkayastha Can you take a look at this pattern of just including the breaking change in the overview doc for breaking changes and then linnking to a dotnet/sdk or dotnet/core issue for the details?

Reply:
Hah, I just noticed at the same time that previously in net 6 there were individual doc pages for each change. That seems better.

Reply:
@nagilson The established process for .NET breaking changes is to open an issue using the breaking change issue template, and then someone from the docs team (me) transcribes that into a doc: https://github.com/dotnet/docs/issues/new?assignees=gewarren&labels=breaking-change%2CPri1%2Cdoc-idea&template=breaking-change.yml&title=%5BBreaking+change%5D%3A+

Reply:
Thanks for pointing me to that. I can go fill that out... there is another breaking change we will likely make in 7.0.200 as well, anywho

Reply:
I will close this since there are now 2 issues filled out using the template. As for the other breaking change, it hasn't been merged yet, and we may be able to circumvent it, so I won't make that one for now.